test/onnx/verify_onnx.cpp

```diff
@@ -1932,6 +1932,28 @@ TEST_CASE(qlinearaveragepool_nt_cip_test)
     EXPECT(migraphx::verify::verify_rms_range(result_vector, gold));
 }
 
```

Comment:
A request: could you please create a test for a 3D tensor concatenation. Axis should be 1. Input types: int8. Output type uint8: 

The Input (two) tensors could be of dimension (just to illustrate) say: 3x4x2 & 3x2x2. Output should be shaped: 3x6x2.  You can keep the input values simple. All same (e.g. 10) in one input tensor. All the same (e.g. 25) in another input tensor. I am just suggesting values just to give an idea, so please write the test the way you like, based on your scale etc. Let the scale of input_1, input_2 and of the output be all different, with different zero points as well. I suggest this, because our testing of the original 'concat' operator are all along axis=0, and we should do more, ideally. If it is already tested somewhere else for the concat, please ignore my suggestion.